packages/react/src/runtimes/core/BaseAssistantRuntimeCore.tsx

```
@@ -1,18 +1,18 @@
 import type { Unsubscribe } from "../../types/Unsubscribe";
 import type { AssistantRuntimeCore } from "./AssistantRuntimeCore";
-import { ProxyConfigProvider } from "../../utils/ProxyConfigProvider";
+import { CompositeContextProvider } from "../../utils/CompositeContextProvider";
 import { ThreadListRuntimeCore } from "./ThreadListRuntimeCore";
 
 export abstract class BaseAssistantRuntimeCore implements AssistantRuntimeCore {
-  protected readonly _proxyConfigProvider = new ProxyConfigProvider();
+  protected readonly _contextProvider = new CompositeContextProvider();
   public abstract get threads(): ThreadListRuntimeCore;
 
   constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)